tests/src/baseservices/threading/ApartmentState/SetApartmentStateDefaultMta.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+public static class SetApartmentStateDefaultMtaTests
+{
+    public static int Main()
```

Comment:
We have tests for the STAThread/MTAThread attributes in CoreFX. They will need to be updated. I do not think we need to duplicate them here.

Reply:
I see, I have updated the CoreFX tests in https://github.com/dotnet/corefx/pull/31682